pkg/serverless/protocol.go

```diff
@@ -273,6 +288,24 @@ func (d *Daemon) WaitUntilClientReady(timeout time.Duration) bool {
 	return d.clientLibReady
 }
 
+// ComputeGlobalTags extracts tags from the ARN, merge them use any user-predefined tags and set them to traces, logs and metrics
+func (d *Daemon) ComputeGlobalTags(arn string, tags []string) {
```

Comment:
What's the casing of functionArn here? Usually, we make sure the tags are lower case, including functionArn.

Reply:
Yes it's lower cased downstream here : https://github.com/DataDog/datadog-agent/pull/8296/files#diff-bf4b276d7656d4b9786de3d61381eaaa8270a2d65b6a49f6f466a473b9dbd7b2R85

Reply:
Could we add a test case with an uppercased functionName to be safe?

Reply:
✅